hive/steem/steem_client.py

```diff
@@ -26,14 +26,11 @@ def instance(cls):
     def __init__(self, url, max_batch=500, max_workers=1):
         assert url, 'steem-API endpoint undefined'
         assert max_batch > 0 and max_batch <= 5000
-        assert max_workers > 0 and max_workers <= 32
+        assert max_workers > 0 and max_workers <= 64
```

Comment:
would it make sense to use env vars for the worker & batch sizes so they can be more easily tune-able?

Reply:
oh these are sanity checks... but why are they set to what they are currently? i.e. why the change to max_workers?

Reply:
Testing revealed that in some cases ~40 client workers were more efficient, and at 64 it can still be competitive (even w/ increased errors). So just widening the range of values considered sane.